### PR TITLE
Home task: HTML 01

### PR DESCRIPTION
**Pull request by Aliaksei888**
1_1 - text Python 1
1_2 - text Python 2
1_3 - text Python 3
1_Simpsons - text Illegible code

### CODE REVIEW

File: AliakseiShas/01_HTML/1_1.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        <p>Введите эти 4 строки:</p>
+<pre>#!/usr/bin/env python
+print 'Hello. I am a python program.'
+name = raw_input("What is your name?")
+print "Hello there, " + name + "!"</pre>
+        <p>Это всё. Сохраните файл как <stronge>hello.py</stronge> в любом месте. Я предлагаю сохранить его в папке
+        <stronge>python_examples</stronge> в домашнем каталоге. Этот пример показывает, как просто написать
+        программу на Python. Для запуска программы нужно   сделать её файл исполняемым. Для
+        этого введите в терминале</p>
+        <p><b>chmod +x hello.py</b></p>
```
Comment (pluhin) [inline AliakseiShas/01_HTML/1_1.html]: it isn't paragraph

File: AliakseiShas/01_HTML/1_2.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		<h2>Списки</h2>>
+		<p>Давайте рассмотрим другой тип переменных, который называется <i>списки</i>. В других языках программирования они называются <i>массивами</i>. Если мы обратимся к аналогии с коробками для обуви, то массив (список) представляет собой несколько склеенных друг с другом коробок с одинаковыми вещами. Например, в одной коробке лежат вилки, в другой — ножи, в третьей — ложки. Посмотрим на простой список— список месяцев. Мы запрограммируем его так:</p>
+		<p><code>months =<Br>
+		['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec']</code></p>
+		<p>Чтобы создать список, мы заключили все значения в квадратные скобки ( '[' и ']' ). Мы назвали наш список <i>'months'</i>. Чтобы использовать список, мы дадим команду print <STRONG>months[0]</STRONG> или <STRONG>months[1]</STRONG> (которые выведут на экран <i>'Jan'</i> или <i>'Feb'</i>). Помните: счет
+		начинается с нуля. Чтобы найти длину списка, можно использовать такую функцию:</p>
+		<p><CODE>print len(months)</CODE></p>
+		<p>которая вернёт 12.</p>>
```
Comment (pluhin) [inline AliakseiShas/01_HTML/1_2.html:29]: extra `>` in the end

File: AliakseiShas/01_HTML/1_2.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+		<h4>Ещё один пример списков — категории в поваренной книге. Например...</h4>>
+<pre>categories = ['Main dish','Meat','Fish','Soup','Cookies']</pre>
+		<p>Таким образом, в <strong>categories[0]</strong> хранится <i>'Main dish'</i>, а в <strong>categories[4]</strong> — <i>'Cookies'</i>. Всё очень просто. Я уверен, вы сможете придумать много других примеров использования списков.</p>
+		<hr>
+		<h2>Комментарии</h2>
+		<p>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам. Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ, комментарии станут важными. Также при их помощи вы заставите Python игнорировать какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'. 
+		Например:</p>>
+		<p>#</p>
```
Comment (pluhin) [inline AliakseiShas/01_HTML/1_2.html]: it isn't paragraph

**Review by pluhin — CHANGES_REQUESTED** [review]

File: AliakseiShas/01_HTML/1_2.html (hunk 2 above)
Comment (Aliaksei888) [reply AliakseiShas/01_HTML/1_2.html:29]: I chenged

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**